.evergreen/config.yml

```diff
@@ -380,13 +380,7 @@ functions:
       params:
         shell: bash
         working_dir: ${working_dir|libmongocrypt}
-        script: |
-          # Authenticate to artifactory.
-          echo "${artifactory_password}" | docker login --password-stdin --username "${artifactory_username}" artifactory.corp.mongodb.com
```

Comment:
The login may still be required for Garasign and Silkbomb, although I don't recall if we have CI tasks that run those.

Reply:
Good point. There is a call of "earthly" Evergreen function that using the Garasign image. The caller authenticates.
Though I expect authenticating may be an easy step to miss if future Evergreen tasks use "earthly" requiring Artifactory. Updated to authenticate as part of the "earthly" Evergreen function, rather than in the caller.